src/components/Home/Home.render.js

```diff
@@ -17,7 +17,7 @@ const HomeRender = ({ match, getSource, ...rest }) => {
   const source = match.params.source ? getSource(match.params.source) : undefined;
   const title = source ? source.title : 'Explore';
 
-  return <Home title={title} source={source} {...rest} />;
+  return <Home title={title} source={source} {...rest} match={match} />;
```

Comment:
we don't need to pass `match` to Home props anymore because I extracted the source id (`source`) from it already in
```javascript
const source = match.params.source ? getSource(match.params.source) : undefined;

return <Home title={title} source={source} {...rest} />;
```

Reply:
maybe you just need to change source to sourceId

```javascript
const sourceId = match.params.sourceId ? getSource(match.params.sourceId) : undefined;

return <Home title={title} sourceId={sourceId} {...rest} />;
```